A/AmazonCACerts/bundled/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020 Invenia Technical Computing Corporation
```

Comment:
This is actually the license for using the certs I believe and so this and the `install_license` line can be removed as well as the `DirectorySource`.